main.lua

```diff
@@ -529,10 +529,9 @@ local function onRender(t)
 	end
 	if EID.isHidden then
 		return
-	elseif EID.Config["HideInBattle"] then
```

Comment:
Why does it remove the code to hide it in battle?

Reply:
> Why does it remove the code to hide it in battle?

I did not intend to do this. Honest mistake, sorry about this!

Reply:
Restored missing code HideInBattle code in current commit.